src/test/java/Page_Objects/GamePage.java

```diff
@@ -12,21 +12,28 @@
 
 
 import java.time.Duration;
+import java.util.Timer;
 
 public class GamePage {
 
     private WebDriver driver;
     private WebDriverWait wait;
     private Actions actions;
+    private Timer timer;
 
     @FindBy(css = "[data-test='button-restart-song']")
     private WebElement restartButton;
 
+    @FindBy(css = "[data-test='button-exit-song']")
+    private WebElement exitButton;
+
+
     public GamePage(WebDriver driver) {
         this.driver = driver;
         PageFactory.initElements(driver, this);
         wait = new WebDriverWait(driver, Duration.ofSeconds(10));
         actions = new Actions(driver);
+        timer = new Timer();

```

Comment:
is this needed

Reply:
No, you're right - I forgot to delete it. Thanks :)